experimental/dds/tree2/src/test/feature-libraries/editable-tree-2/proxies.spec.ts

```diff
@@ -4,23 +4,73 @@
  */
 
 import { strict as assert } from "assert";
+import { SchemaBuilder } from "../../../domains";
 import { is, typeNameSymbol } from "../../../feature-libraries";
-import { leaf, SchemaBuilder } from "../../../domains";
 import { itWithRoot } from "./utils";
 
+describe("SharedTreeObject factories", () => {
```

Comment:
generally we have a test file for each source files, so this would go over in a schemaBuilder.spec.ts file

Reply:
Good idea, I'll do that.